examples/02-Merlin-Models-and-NVTabular-applying-to-your-own-dataset.ipynb

```diff
@@ -7,7 +7,7 @@
    "metadata": {},
```

Comment:
<p>Why do we add "name+"_02"" "_02" to the path?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>The reason was that the 1st example notebook was also saving to the same output path, potentially causing some. For example, in example notebook 2 I accidentally skip the step that runs the NVT workflow, and then in the models part of the notebook I end up using the dataset preprocessed on notebook example 1 (saved in the same path), which has different features than the ones expected.</p>

Reply:
<p>I think we can assume that the user runs the notebooks end-to-end and do not skip code cells. If that is the case, we do not need to capture the edge case? I think there could be many breaking points, if users change code / do not run all cells</p>

Reply:
I just think users can potentially be skipping some cell when having multiple example notebooks open when getting to know the library. That mistake happened to me and took a few minutes for me to figure it out that I was using cached results from other notebook. We'd better avoiding that with this simple change (outputing preprocessed files in different folders for different examples).